### ISSUE TITLE
Can capsa be used on pretrained models?

### ISSUE THREAD

**Issue by AanchalA** [themis-ai/capsa#119]

**Comment by egesko** [comment]
Almost all capsa wrappers support pretrained models except `EnsembleWrapper()`. That particular wrapper randomizes model weights to make sure that each ensemble can converge towards different optimums in loss function, giving us an estimate of model uncertainty.

**Comment by AanchalA** [comment]
Can you provide a code based example for bias detection using a pre-trained model?

**Comment by egesko** [comment]
Closing the issue as solved. Feel free to open a new one if you have any further questions.
